client/lib/login/index.js

```diff
@@ -248,7 +248,7 @@ export const getLoginLinkPageUrl = ( {
 	return login( loginParameters );
 };
 
-export const getPluginTitle = ( pluginName, translate ) => {
+export const getPluginTitle = ( pluginName, translate, langSlug = 'en' ) => {
```

Comment:
I wonder if it's more useful for the locale to be passed to this function, instead of calling `getLocaleSlug` directly in it? IMO it's simpler to call it directly, but I may have missed something!

Reply:
Or alternatively, we could change it to `langSlug = getLocaleSlug()`, and remove the parameter in all consumers to simplify it

Reply:
@ilyasfoo I like the 2nd one! I'll make an update.